apps/src/applab/RecordListener.js

```diff
@@ -26,7 +26,7 @@ var RecordListener = module.exports = function () {
  * processing the last batch of data for that table.
  * @type {number}
  */
-var RECORD_INTERVAL = 1000; // 1 second
+var RECORD_INTERVAL = 3000; // 3 seconds
```

Comment:
Just a caution, 3 seconds is still pretty short.  Even with Pusher, [NetSim uses a 10-second default polling interval](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/NetSimTable.js#L23) and requires a minimum of 2.5 seconds between reads.  I think when NetSim was just doing polling, we usually kept it around 5 seconds.